src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts

```diff
@@ -12,26 +12,21 @@ describe('Target Page visualization boxes', () => {
     let targetPage: TargetPage;
     let popupPage: PopupPage;
 
-    beforeAll(async () => {
+    beforeEach(async () => {
         browser = await launchBrowser({ suppressFirstTimeDialog: true });
         targetPage = await browser.newTargetPage();
         popupPage = await browser.newPopupPage(targetPage);
         await popupPage.gotoAdhocPanel();
     });
 
-    beforeEach(async () => {
-        await popupPage.disableAllToggles();
-        await targetPage.waitForVisualizationBoxesToDisappear();
-    });
```

Comment:
Are `disableAllToggles()` and `waitForVisualizationBoxesToDisappear()` used elsewhere? If not, should we remove the unused code?

Reply:
I will check and update this.

Reply:
Done. Thanks for catching that.